### PR TITLE
Centering Title and badges

### PR DESCRIPTION
**Pull request by DEBSUBHRO**
Issue #79

Changes
Centering title and badges.

Related Issues
Issue #79
Pull Request #269

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,12 +4,22 @@
...
+  </a>
+  <a href="https://lernajs.io">
+    <img src="https://img.shields.io/badge/maintained%20with-lerna-cc00ff.svg" alt="lerna"/>
+  </a>
+  <a href="https://accord-project-slack-signup.herokuapp.com">
+    <img src="https://img.shields.io/badge/Slack-Join%20Slack-blue" alt="join slack"/>
+  </a>
+</p>
 
 Repository for Smart Legal Contract Templates that conform that to the [Accord Project Template Specification][techspec], the protocol is managed by the open-source community of the [Accord Project][aphome].
```
Comment (irmerk) [inline README.md]: Something is happening here where this is a `h1`

**Comment by irmerk** [comment]
Also, you'll want to make sure to do a couple more things to the PR as well:

- Fill out the PR more (the issues and PRs your referencing aren't correct), you can reference [other merged PRs here](https://github.com/accordproject/cicero-template-library/pulls?q=is%3Apr+is%3Aclosed).
- Add a DCO (Developers Certificate of Originality) per [the instructions](https://github.com/accordproject/cicero-template-library/blob/master/CONTRIBUTING.md):
> Ensure you provide a DCO sign-off for your commits using the -s option of git commit. For more information see [how this works](https://github.com/probot/dco#how-it-works).

**Comment by DEBSUBHRO** [comment]
Pulled the request #271. Kindly check it.
Thank you.

**Comment by irmerk** [comment]
> Pulled the request #271. Kindly check it.
> Thank you.

This is a PR from someone else.

**Pull request closed without merging**